packages/core/src/customizations/components/overrides/ThirdPartyScripts.tsx

```diff
@@ -0,0 +1,36 @@
+import { default as ThirdPartyScripts } from 'src/customizations/scripts/ThirdPartyScripts'
+
+const RenderThirdPartyScripts = (
+  ThirdPartyScripts: () => string[] | string
+) => {
+  const scriptContent = ThirdPartyScripts() || []
+
+  const buildScript = (script: string, index?: number) => (
+    <script
+      key={`thirdpartyscript-${index}`}
+      type="text/javascript"
+      dangerouslySetInnerHTML={{
+        __html: script,
+      }}
+    />
+  )
+
+  const renderScriptTags = (): JSX.Element => {
+    if (Array.isArray(scriptContent)) {
+      return (
+        <>{scriptContent.map((script, index) => buildScript(script, index))}</>
+      )
+    }
+    return buildScript(scriptContent)
+  }
+
+  return renderScriptTags
+}
+
+const overrides = {
+  components: {
+    ThirdPartyScripts: RenderThirdPartyScripts(ThirdPartyScripts),
```

Comment:
I don't think the `RenderThirdPartyScripts` logic is the way to go. I'd rather we just receive components from the customization and then render them.